src/enkf/gridio_fv3reg.f90

```diff
@@ -1024,10 +1016,7 @@ subroutine writegriddata(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,vargrid
                 enddo
              enddo
              workvar3d=workvar3d+workinc3d
-             if ( l_use_enkf_directZDA .and. cliptracers ) then ! set cliptracers to remove negative hydrometers
-                 clip = tiny(workvar3d(1,1,1))
-                 where (workvar3d < clip) workvar3d = clip
-             end if
+             if (  cliptracers ) where (workvar3d < clip) workvar3d = clip
```

Comment:
Nitpick:  remove extra space before `cliptracers` in `if` statement.   That is.  instead of 

              workvar3d=workvar3d+workinc3d
              if (  cliptracers ) where (workvar3d < clip) workvar3d = clip
              call write_fv3_restart_data3d(varstrname,fv3filename,file_id,workvar3d)

have

              workvar3d=workvar3d+workinc3d
              if ( cliptracers ) where (workvar3d < clip) workvar3d = clip
              call write_fv3_restart_data3d(varstrname,fv3filename,file_id,workvar3d)